docs/ops/movement/Pad_1.md

```diff
@@ -122,4 +126,35 @@ OUTPUT =
         </port>
     </output>
 </layer>
+```
+
+**Example**: edge mode
+
+```xml
+<layer ... type="Pad" ...>
+    <data pad_mode="edge"/>
+    <input>
+        <port id="0">
+            <dim>1</dim>
+            <dim>3</dim>
+            <dim>32</dim>
+            <dim>40</dim>
+        </port>
+        <port id="1">
+            <dim>4</dim>     <!-- pads_begin = [0, 5, 2, 1]  -->
+        </port>
+        <port id="2">
+            <dim>4</dim>     <!-- pads_end = [1, 0, 3, 7] -->
+        </port>
+    </input>
+    <output>
+        <port id="0">
+            <dim>2</dim>     <!-- 2 = 0 + 1 + 1 = pads_begin[0] + input.shape[0] + pads_end[0] -->
+            <dim>8</dim>     <!-- 8 = 5 + 3 + 0 = pads_begin[1] + input.shape[1] + pads_end[1] -->
+            <dim>37</dim>    <!-- 37 = 2 + 32 + 3 = pads_begin[2] + input.shape[2] + pads_end[2] -->
+            <dim>48</dim>    <!-- 48 = 1 + 40 + 7 = pads_begin[3] + input.shape[3] + pads_end[3] -->
+                             <!-- all new elements are filled with 15.0 value -->
```

Comment:
Why does the elements filled with 15.0 for the edge Pad?

Reply:
It won't be fill by `15`. Fix: https://github.com/openvinotoolkit/openvino/pull/5211